geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/LauncherLifecycleCommands.java

```diff
@@ -771,11 +771,18 @@ public Result statusLocator(
               CliStrings.STATUS_SERVICE__GFSH_NOT_CONNECTED_ERROR_MESSAGE, LOCATOR_TERM_NAME));
         }
       } else {
-        final LocatorLauncher locatorLauncher =
-            new LocatorLauncher.Builder().setCommand(LocatorLauncher.Command.STATUS)
-                .setBindAddress(locatorHost).setDebug(isDebugging()).setPid(pid)
-                .setPort(locatorPort).setWorkingDirectory(workingDirectory).build();
+        final LocatorLauncher locatorLauncher;
 
+        if ((locatorHost == null) && (locatorPort == null) && (workingDirectory == null)) {
```

Comment:
I had thought that `locatorPort` would not uniquely identify a locator without also specifying a `locatorHost`, but it looks like we might end up falling back to a default value of localhost.  It would be nice to add a test that make sure this works as expected.  (I can pair with you on that if you'd like!)